src/components/CardPromo/CardPromo.tsx

```diff
@@ -0,0 +1,96 @@
+import { HTMLAttributes, useState } from "react";
+import styled from "styled-components";
+import { Icon, IconName, Text } from "@/components";
+
+export interface CardPromoProps extends HTMLAttributes<HTMLDivElement> {
+  label: string;
+  icon: IconName;
+  dismissible?: boolean;
+}
+const Background = styled.div`
+  ${({ theme }) => `
+  background-image: ${theme.click.card.promotion.color.stroke.default};
```

Comment:
Should we use the stroke for the background image?
Is this intended?
Should we rename it to something else?

Reply:
It's intended, turns out my bizarre design is very hard to implement in CSS as there is no linear-gradient option on borders, most workarounds result in the border radius stops working. 

So this fairly simple workaround is to have a div behind the div with a padding of 1px. This way the background image pokes out around the side acting as a gradient border. So I think the token name is correct even if the implementation is a little unorthodox.